content/docs/user-guide/editor/entity-outliner.md

```diff
@@ -15,5 +17,92 @@ The **Entity Outliner** shows all the entities and prefabs in the level. You can
 
    You can select multiple entities by doing one of the following in the **Entity Outliner**:
 
-   * Use **Ctrl + Left Mouse Button** to select multiple entities one at a time to be part of a multi-selection.
-   * Use **Shift + Left Mouse Button** to select a range of entities.
+- Use **Ctrl + Left Mouse Button** to select multiple entities one at a time to be part of a multi-selection.
+- Use **Shift + Left Mouse Button** to select a range of entities.
+
+## Reordering Entities
+
+When you create entities or instantiate prefabs, they appear at the bottom of the list in the **Entity Outliner**.
```

Comment:
At some point it might be worth mentioning about selection rules, as if you create an entity with an entity selected, it will be parented under the selected entity. Perhaps you could say 'When you create entities or prefabs from Entity Outliner, they....' to be more specific?

Reply:
Updated the content to cover both cases.

Reply:
That's great thanks!